src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -45,6 +52,11 @@ export default class DropDownComponent extends ComponentBase<azdata.DropDownProp
 	private _selectBox: SelectBox;
 	private _isInAccessibilityMode: boolean;
 	private _loadingBox: SelectBox;
+	public isValid: boolean;
+	/**
+	 * This flag is used to hide the error message in the initial state of the dropdown. We do not want to show the error message before user has interacted with it.
+	 */
+	public isInitState: boolean = true;
```

Comment:
do you really need this flag? if the default value for isValid is true then you don't need this, right?

Reply:
The validation is run when the dropdown is first initialized. If the dropdown is empty, the validation will fail, and error will be shown